wandb/sdk/launch/runner/vertex_runner.py

```diff
@@ -59,12 +60,14 @@ def wait(self) -> bool:
 
     def get_status(self) -> Status:
         job_state = str(self._job.state)  # extract from type PipelineState
-        if job_state == "PipelineState.PIPELINE_STATE_SUCCEEDED":
+        if job_state == "JobState.JOB_STATE_SUCCEEDED":
             return Status("finished")
-        if job_state == "PipelineState.PIPELINE_STATE_FAILED":
+        if job_state == "JobState.JOB_STATE_FAILED":
             return Status("failed")
-        if job_state == "PipelineState.PIPELINE_STATE_RUNNING":
+        if job_state == "JobState.JOB_STATE_RUNNING":
             return Status("running")
+        if job_state == "JobState.JOB_STATE_PENDING":
```

Comment:
I'm confused about this change, is this a change in vertex? Or are we using a different way to submit jobs?

Reply:
This is a change to the status codes made by vertex https://cloud.google.com/vertex-ai/docs/reference/rest/v1/JobState